_sass/mixin.scss

```diff
@@ -37,13 +39,49 @@
       line-height: 2;
     }
   }
+  ul.tabs-vertical {
+      li {
+          cursor: pointer;
+          border: none;
+          list-style-image: none;
+          background: $white;
+          border-radius: 4px 4px 0 0;
+          margin: 0 6px 0 0;
+          padding: 0 10px;
+          border-bottom: 3px solid $white;
+          a, a:visited {
+              padding: 0;
+              font-weight: normal;
+              text-decoration: none;
+              color: $contentColor;
+              font-size: 90%;
+              background: transparent;
+          }
+    }
+        li.active, li.ui-tabs-active {
```

Comment:
Would you please check all the indentations in your changes and make corrections where needed?
Thanks.